R/use_r_workflows.R

```diff
@@ -253,7 +253,10 @@ use_doc_and_style_r <- function(workflow_name = "call-doc-and-style-r.yml",
   usethis::use_git_ignore(ignores = "*.rds", directory = file.path(".github"))
 }
 
-#' use workflow in current pkg to update pkg down, where the site is deployed to a branch called gh-pages
+#' Creates a workflow in current R package to update an existing pkgdown GitHub pages site
+#' For more information, see [the setup details vignette](https://nmfs-fish-tools.github.io/ghactions4r/articles/set_up.html) before
+#' running this function. This workflow will assume the webiste is built form

```

Comment:
minor typo: change "webiste" to "website" and change "built form" to "built from".

Reply:
good catch, thank you!!